UPGRADE.md

```diff
@@ -1,3 +1,10 @@
+# Upgrade to 2.6
```

Comment:
I think we need to keep this for 3.6, as we don't have a way to stop users from upgrading...